core-java-modules/core-java-21/src/test/java/com/baeldung/java21/EmojiSupportUnitTest.java

```diff
@@ -0,0 +1,73 @@
+package com.baeldung.java21;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.junit.jupiter.api.Test;
+
+class EmojiSupportUnitTest {
+
+    @Test
+    void when_message_contains_emoji_then_isEmoji_returns_true() {
```

Comment:
The BDD test name format should be like this:
whenMessageContainsEmoji_thenIsEmojiReturnsTrue

(same for other tests)

Reply:
fixed!